lib/ota/common.js

```diff
@@ -430,6 +430,7 @@ async function getNewImage(current, logger, device, getImageMeta, downloadImage)
     assert(image.header.fileVersion === meta.fileVersion, 'File version mismatch');
     assert(!meta.fileSize || image.header.totalImageSize === meta.fileSize, 'Image size mismatch');
     assert(image.header.manufacturerCode === current.manufacturerCode, 'Manufacturer code mismatch');
+    assert(image.header.manufacturerName === current.manufacturerName, 'Manufacturer name mismatch');
```

Comment:
This check should be removed,  the `image.header` never contains a `manufacturerName`

Reply:
OK, everything is clear.